### DISCUSSION TITLE
Order of Highlights

### DISCUSSION THREAD

**Discussion by itsgareth** [community/community#19554] [Other Feature Feedback, Questions, & Ideas]
I know this isn't really a big deal but i personally believe that  the `Pro` highlight should always be displayed bellow all others.
I just looks out of place in its current location.

The option to reorder highlights and achievements would be good feature.

On the left is the current layout, the right is the modified layout
<div align="center">
<img width="321" alt="Screenshot 2022-06-27 at 14 03 35" src="https://user-images.githubusercontent.com/34524710/175948805-1b922305-f0a1-435a-84e7-322926b74b7f.png">

<img width="322" alt="Screenshot 2022-06-27 at 14 02 22" src="https://user-images.githubusercontent.com/34524710/175948830-e15661c6-13d0-4698-80d2-4c7681d079f2.png">
</div>

**Comment by melvincwng** [comment]
@gwdawson Hi Gareth, the modified layout does indeed look neater and more aesthetically pleasing UI-wise.

Great suggestion 👌

Quick question - did you use Photoshop to do the edits? Just curious haha 😆

**Reply by itsgareth** [reply]
nope, just inspect and changed the order of the html elements

**Reply by crizmo** [reply]
For some reasons i dont see that highlights tab on my profile even tho i have pro and developer member enabled
but i do see developer program member from github mobile app
